test/forms_test.js

```diff
@@ -563,9 +563,9 @@ describe('Forms', function() {
       before(function(done) {
         const field1 = browser.querySelector('#field-scary');
         const field2 = browser.querySelector('#field-notscary');
-        browser.choose(field1);
-        field2.addEventListener('focus', ()=> done());
         browser.choose(field2);
+        field1.addEventListener('focus', ()=> done());
+        browser.choose(field1);
       });
```

Comment:
Is this also a case where it calls `done` twice because there's already an event listener registered in an earlier test?